Bottom: 8010ff2901c38d10212e4281db08ce1c676ed0eb
Top:    b7be02c28b0a823e4febc6d0fe6280aca0572320
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:49:21 +0100

Refresh of patch

---

diff --git a/src/draw.c b/src/draw.c
index 2afcf98..10da1dd 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -196,15 +196,6 @@ Draw (void)
 }
 
 /* ---------------------------------------------------------------------------
- * redraws the output area without clearing it
- */
-void
-RedrawOutput (BoxTypePtr area)
-{
-  Redraw ();
-}
-
-/* ---------------------------------------------------------------------------
  * redraws the output area after clearing it
  */
 void
